docs/dasharo-tools-suite/documentation.md

```diff
@@ -34,6 +34,39 @@ to use.
 
 This section describes how to boot DTS using iPXE.
 
+#### Temporary Suspension of Network Booting for DTS
+
+We wish to inform you of an important security concern that has been identified
+in the Dasharo Tools Suite (DTS). As detailed in [Issue
+450](https://github.com/Dasharo/dasharo-issues/issues/450) on our GitHub
+repository, it has come to our attention that the DTS is currently being
+downloaded via HTTP when booting through iPXE. This method, unfortunately,
+exposes users to potential man-in-the-middle attacks due to the lack of
+encryption and security features inherent in HTTP.
+
+##### What This Means for You
+
+* The use of HTTP instead of HTTPS for downloading DTS poses a significant
+  security risk, potentially allowing unauthorized access or alteration of the
+  DTS during download.
+* As a precaution, we strongly advise against using the network boot feature
+  for DTS until this issue is resolved.
+
+##### Our Immediate Actions
+
+* We are actively working on resolving this issue by implementing HTTPS for DTS
+  downloads. Our team is committed to ensuring the security and integrity of
+  our software.
+* Until a fix is deployed, we recommend users to manually download the DTS
```

Comment:
The fix was deployed today on the server side, which makes the https possible. This is one part of the eqation.

The other is, the older releases have embedded ipxe script with http.

It will be changed for https for the following releases:
- NovaCustom 12th Gen 1.7.2 and newer
- NovaCustom 11th Gen 1.5.2 and newer
- MSI Z690 1.1.3 and newer
- MSI Z790 0.9.1 and newer.

For older releases, we can still boot via https, but it would need to be done manually from iPXE shell:

```
dhcp
chain https://boot.dasharo.com/dts/dts.ipx
```

Reply:
So I guess we should recommend ugprading to these versions via USB or via ipxe shell by manually executing the HTTPS URL, and add it to the newsletters? @BeataZdunczyk CC

Reply:
@macpijan @BeataZdunczyk Yes, we should recommend a different update procedure, and the "old method" can be used since the versions mentioned above.

The key question is what to change in documentation.

Reply:
> For older releases, we can still boot via https, but it would need to be done manually from iPXE shell:
> 
> ```
> dhcp
> chain https://boot.dasharo.com/dts/dts.ipxe
> ```

Assuming that certificate chain is not too long and is not fragmented. Older releases may not have the most up to date iPXE version

Reply:
Another thing is that some people may already have 1.7.1 with FUM, which will automatically boot DTS via HTTP, so it must not be used. So one has to disable locks manually.

Reply:
https://github.com/Dasharo/docs/pull/727/commits/a1964d42cca1392ad14e749cc106c08714202641

Reply:
@pietrushnic @BeataZdunczyk open for comments on above commit

Reply:
Great, I approve @miczyg1

Reply:
@miczyg1, your changes are ok. I will resolve conflicts and push for an improved version. I guess we can resolve this conversation and merge after you approve of changes resolving conflict.